src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -3028,6 +3082,10 @@ TEST_F(fully_connected_gpu_tests, compressed_scale_fp16_cached) {
     this->test_compressed_scale_fp16(false);
 }
 
+TEST_F(fully_connected_gpu_tests, compressed_int8_scale_zp_scalar) {
```

Comment:
nit: the purpose of this test is unclear from the name. what about adding a comment for it?